Core/Graphics/Source/GraphicsImpl.cpp

```diff
@@ -54,30 +64,28 @@ namespace Babylon
         UpdateBgfxResolution();
     }
 
-    void Graphics::Impl::UpdateBgfxResolution()
+    void Graphics::Impl::AddToJavaScript(Napi::Env env)
     {
-        std::scoped_lock lock{m_state.Mutex};
-        m_state.Bgfx.Dirty = true;
-        auto& res = m_state.Bgfx.InitState.resolution;
-        auto level = m_state.Resolution.HardwareScalingLevel;
-        res.width = static_cast<uint32_t>(m_state.Resolution.Width / level);
-        res.height = static_cast<uint32_t>(m_state.Resolution.Height / level);
+        JsRuntime::NativeObject::GetFromJavaScript(env)
+            .Set(JS_GRAPHICS_NAME, Napi::External<Impl>::New(env, this));
     }
 
-    void Graphics::Impl::AddRenderWorkTask(arcana::task<void, std::exception_ptr> renderWorkTask)
+    Graphics::Impl& Graphics::Impl::GetFromJavaScript(Napi::Env env)
     {
-        std::scoped_lock RenderWorkTasksLock{m_renderWorkTasksMutex};
-        m_renderWorkTasks.push_back(std::move(renderWorkTask));
+        return *JsRuntime::NativeObject::GetFromJavaScript(env)
+                    .Get(JS_GRAPHICS_NAME)
+                    .As<Napi::External<Graphics::Impl>>()
+                    .Data();
```

Comment:
This seems extra extra tabbed.
```suggestion
            .Get(JS_GRAPHICS_NAME)
            .As<Napi::External<Graphics::Impl>>()
            .Data();
```

Reply:
This is the behavior of clang-format. The idea is that the indent is from the statement (not including the return keyword).